graphql/quizzes/upsertCodingLanguageQuiz.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "@apollo/client";
+
+export const UPSERT_CODING_LANGUAGE_QUIZ_RESPONSE = gql`
+  mutation upsertCodingLanguageQuizResponse(
+    $objects: [coding_language_quiz_insert_input!]!
```

Comment:
are you allowing for multiple quiz responses to be inserted at once? Is there a way to simplify this query so you're only adding one row as needed. If you can't come up with a cleaner query, that's ok. It might not be possible

Reply:
Separated the logic. We now have an insert and update query!